python/lsst/dax/imgserv/vo/schemas/VOSIAvailability-v1.0.xsd

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsd:schema targetNamespace="http://www.ivoa.net/xml/VOSIAvailability/v1.0"
```

Comment:
Where do we use these .xsd files?

Reply:
These schemas, behind the XML responses being served, are gathered from IVOA site, to be kept for our local reference, as well as preparing for future validation (programmatically when ready). Today, that can't be done yet, e.g. VOMetadata-v0.1.xsd (404 Not Found).